src/features/search/components/__tests__/useLocationChoice.test.ts

```diff
@@ -0,0 +1,90 @@
+import { useLocationChoice } from 'features/search/components/useLocationChoice'
+import { LocationType } from 'features/search/enums'
+import { initialSearchState } from 'features/search/pages/reducer'
+import { MAX_RADIUS } from 'features/search/pages/reducer.helpers'
+import { GeoCoordinates } from 'libs/geolocation'
+import { SuggestedPlace } from 'libs/place'
+import { SuggestedVenue } from 'libs/venue'
+import { mockedSuggestedVenues } from 'libs/venue/fixtures/mockedSuggestedVenues'
+import { BicolorAroundMe as AroundMe } from 'ui/svg/icons/BicolorAroundMe'
+import { BicolorEverywhere as Everywhere } from 'ui/svg/icons/BicolorEverywhere'
+import { BicolorLocationBuilding as LocationBuilding } from 'ui/svg/icons/BicolorLocationBuilding'
+import { BicolorLocationPointer as LocationPointer } from 'ui/svg/icons/BicolorLocationPointer'
+
+let mockSearchState = initialSearchState
+jest.mock('features/search/pages/SearchWrapper', () => ({
+  useSearch: () => ({
+    searchState: mockSearchState,
+  }),
+}))
+
+const DEFAULT_POSITION = { latitude: 2, longitude: 40 } as GeoCoordinates
```

Comment:
C'est un détail, mais c'est mieux de faire
```suggestion
const DEFAULT_POSITION: GeoCoordinates = { latitude: 2, longitude: 40 }
```

puisque le keyword `as` est surtout utilisé pour override un type.